fleet/views.py

```diff
@@ -8,11 +8,7 @@ class DriverDetails(RetrieveAPIView):
 
     permission_classes = [IsAuthenticated]
     serializer_class = DriverSerializer
+    queryset=Driver.objects.all()
 
-    def get(self, request, *args, **kwargs):
-
-        id=request.user.id  
-        driver=Driver.objects.get(user__id=id)
-
-        serializer=DriverSerializer(driver)
-        return Response(serializer.data)
\ No newline at end of file
+    def get_object(self):
+        return Driver.objects.get(user=self.request.user)
```

Comment:
What if object not found? 
Add Try Catch i point out this second time

Reply:
done